### DISCUSSION TITLE
Custom auth page design

### DISCUSSION THREAD

**Discussion by boris-glumpler** [filamentphp/filament#7710] [Help] [answered]
What's the best way to implement a custom design for the auth pages in v3 to match the front end? Just CSS won't work for me, unfortunately. 

It looks like I'll have to override each auth page and set a custom view which then uses my custom layout. Ideally, I'd like to specify this layout template directly in the panel options. Maybe something for later?

Cheers!

**Comment by danharrin** [comment, marked as answer]
Extend the `Login` class, copy the original `.blade.php` file into your project, set the `$view` property on the class, and pass the new class to `->login()` in the panel provider

**Reply by boris-glumpler** [reply]
Yup, that's what I thought. Cheers!
